Classes/Extensions/Amazing+Tools.swift

```diff
@@ -27,9 +27,9 @@ public extension Amazing {
 }
 
 
-extension Awesome.brand {
+extension Awesome.brands {
```

Comment:
I think I’d like to keep just brand as brands would imly you could select more then one behind the next dot? What do you think?

Reply:
The reason I changed it to `brands` is that on the Font Awesome Cheatsheet the brand icons are marked with the id `brands` and not `brand`.
The `CheatSheetParser` essentially has no idea which keys exist and simply maps all icons into an array of dictionaries with the corresponding key e.g. `brands`.
But it would be easy to change the key while generating the enums, with a simple condition.
Your call 😄.